### DISCUSSION TITLE
TAPO C425 - Not Streamig

### DISCUSSION THREAD

**Discussion by dwdonnelly** [JurajNyiri/HomeAssistant-Tapo-Control#787] [Q&A]
Happy Holidays All.

I have successfully installed the HomeAssistant-Tapo-Control integration.  I also have the TP-Link integration installed months ago.

The Tapo-Control integration immediately saw the Tapo C425 camera and all the device parameters appeared.   Well - all except for the live video/stream.

Every time I attempt to configure the camera for the live feed, it tells me "authentication error".  I have verified the user name/pwd and IP address.

In further researching this problem, https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/discussions/646#discussioncomment-11496912,  appears the C425 is accessible via HA or this integration.

Is there a workaround to get the video stream?  Or this camera, C425, simply does not provide that ability?

**Comment by bebe1310** [comment]
It seems to me that the live is not possible on battery cameras. Am I wrong?

**Reply by dwdonnelly** [reply]
@bebe1310 Well - I am not sure about that battery aspect, but my research has led me to here:  https://www.youtube.com/watch?v=vVDLc4BbJuk.  Tech Man Pat shows the C425 installed and operating via HA.  Now, is that because of the solar panel attached vice no solar panel?  I posted a comment to him asking for his help to understand how he achieved the integration.  Will update as I learn more.

Have others seen this and know how to integrate the C425?  Very open to all suggestions.  As always - many thanks in advance!

**Reply by bebe1310** [reply]
Didn't see HA in this vidéo, only tapo app
Sorry
I have c425 and I didn't succès to have vidéo in HA. Only use some paramèters on automatisations like alarms
I read that it's because it's a battery camera. To be confirmed by another people

**Reply by dwdonnelly** [reply]
@bebe1310 WOW!  You are correct.  It looked so much like HA.....WOW!  Thanks!  Will keep researching.  As always, any and all help is appreciated.

**Reply by iyademowo** [reply]
Tapo's argument is that because it's battery powered, RTSP cannot be on it... since RTSP needs consistent stream and the camera being battery powered goes into hibernation until the sensors gets active and it streams.... Only that it was till a firmware upgrade took it out, so it don't understand the argument.
While it may make sense to conserve battery (which is another matter because they lied about that 300 days marker), the customer should be able to decide how they intend to use their product. Only have a caveat that it would drain faster if run in this way. 
What is the point of the Solar panels if you cannot push it as far as you please? I had to put AC to mine.... had a smart socket to turn on the power to it when i need to charge... and trust me, i do that at least once a month... horrible.

I wouldn't mind the false battery spec if I had RTSP. I have found a way to solve that... but no battery that can last the time stated, no RTSP to use as I please... I think i will be dumping this. Even my lower C121s have camera account and you can enjoy it... 

TAPO NEEDS TO DO BETTER... and it's not too late to retrace their steps and return RTSP to C425 (Yes, with a caveat that we are responsible for battery drain). Not doing that, you can be guaranteed the cameras would not stay up for too long anyway. The world has moved past locking us up like this.
